donut_shop.html

```diff
@@ -19,33 +19,26 @@
     <td>3:00 AM </td>
     <td>4:00 AM</td>  
     <td>5:00 PM </td>
-    <td>6:00 PM </td>
     <td>Total </td>
-  <tr id='Downtown'>
-    <th>Downtown</th>
-    <td> </td>
-  </tr>
-<tr id='Capitol Hill'>
-  <th>Capitol Hill</th>
-  <td> </td>
-  </tr>
-<tr id='South Lake Union'>
-  <th>South Lake Union</th>
-  <td> </td>
-  </tr>
-<tr  id='Wedgewood'>
-  <th>Wedgewood</th>
-  <td> </td>
-  </tr>
-<tr id='Ballard'>
-  <th>Ballard</th>
-  <td> </td>
-  </tr>
+  <tbody id="bodyTable">
+  </tbody>
 
 </table>
+ <form id="donutForm">
+      <label for="location">Location</label>
+      <input id="location" type="text">
```

Comment:
You could also use Placeholder attribs in your input tags to help the user understand what type of data to put into their inputs.